facade/servicepath_cache.go

```diff
@@ -27,10 +27,11 @@ type serviceCache struct {
 }
 
 type servicePath struct {
-	serviceID   string
-	tenantID    string
-	parentID    string
-	servicePath string
+	serviceID       string
+	tenantID        string
+	parentID        string
+	servicePath     string
+	serviceNamePath string
```

Comment:
Please update the unit tests in servicepath_cache_test.go to also verify the contents for serviceNamePath